apps/api/src/persistence/repositories/repository.provider.ts

```diff
@@ -56,6 +59,10 @@ export class RepositoryProvider
         return new ArangoCategoryRepository(this.databaseProvider);
     }
 
+    getIdRepository(): IIdRepository<AggregateId> {
```

Comment:
Make sure this is part of the `IRepositoryProvider` interface.

Reply:
I also heavily renamed \ refactored the repository interfaces part of the code base (in the domain).